MyLibrary/BusinessLogicTests/LibraryServiceTests.cs

```diff
@@ -0,0 +1,476 @@
+// <copyright file="LibraryServiceTests.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using System;
+using System.Linq;
+using BusinessLogic.DataProvider;
+using BusinessLogic.LibraryModel;
+using BusinessLogic.LibraryService;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessLogicTests
+{
+    /// <summary>
+    /// Tests for Library Services
+    /// </summary>
+    [TestClass]
+    public class LibraryServiceTests
+    {
+        /// <summary>
+        /// The data provider
+        /// </summary>
+        private static IDataProvider dataProvider;
+
+        /// <summary>
+        /// The instance of Library service for testing
+        /// </summary>
+        private static ILibraryService libraryService;
+
+        /// <summary>
+        /// Initializes test data
+        /// </summary>
+        /// <param name="testContext">The test context</param>
+        [ClassInitialize]
+        public static void TestInitialize(TestContext testContext)
+        {
+            // Arrange
+            dataProvider = new ListsDataProvider();
+            libraryService = new LibraryService(dataProvider);
+        }
+
+        #region BookTests
+
+        /// <summary>
+        /// Tests the method GetBookById
+        /// Correct input values
+        /// Returned correct book
+        /// </summary>
+        /// <param name="id">The book identifier</param>
+        /// <param name="name">The book name</param>
+        /// <param name="year">The publishing year</param>
+        /// <param name="authorId">The author identifier</param>
+        [TestMethod]
+        [DataRow(4u, "����� ��������", 1955, 2u)]
+        public void GetBookByIdTest_CorrectId_ReturnedCorrectBook(uint id, string name,
```

Comment:
Some bad things with russian letters. Try to use English.

Reply:
Very interesting. This case appears only in tests

Reply:
@Anastasiia37 yeah in any way try to avoid russian and use english